packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -157,53 +156,44 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
   }), [state, domRef, direction, collator]);
 
   let provider = useProvider();
-  let {checkboxProps} = useGridSelectionCheckbox({key: null}, state);
   let dragState: DraggableCollectionState;
   if (isListDraggable) {
     dragState = dragHooks.useDraggableCollectionState({
       collection: state.collection,
       selectionManager: state.selectionManager,
       renderPreview(selectedKeys, draggedKey) {
         let item = state.collection.getItem(draggedKey);
+        let isDraggingMultiple = selectedKeys.size > 1;
```

Comment:
lets rename selectedKeys to draggingKeys so it's more apparent that this isn't the selectedKeys of the entire ListView collection

Reply:
good call, fixed this